TWTValidation/Errors/TWTValidationErrors.h

```diff
@@ -82,6 +82,17 @@ extern NSString *const TWTValidationValueValidationErrorsKey;
 extern NSString *const TWTValidationKeyValuePairValidationErrorsKey;
 
 /*!
+ @abstract The error domain for errors originating from parsing the JSON Schema against which JSON is to be validated.
```

Comment:
The error domain for errors originating from TWTValidation’s JSON schema parser.